reco_utils/dataset/sparse.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+Generate the user/item affinity matrix from a pandas dataframe and vice versa
+"""
+
+import pandas as pd
+import numpy as np
+import itertools
+
+from scipy.sparse import coo_matrix
+import logging
+
+# import default parameters
+from reco_utils.common.constants import (
+    DEFAULT_USER_COL,
+    DEFAULT_ITEM_COL,
+    DEFAULT_RATING_COL,
+    DEFAULT_TIMESTAMP_COL,
+    PREDICTION_COL,
+)
+
+# for logging
+log = logging.getLogger(__name__)
+
+
+class AffinityMatrix:
+    """
+
```

Comment:
Good to add what this class is about here, and move the Args description under `__init__`

Reply:
Also seems like this class is not being used from anywhere. Make sure if this is deprecated module or not.